src/Group.cpp

```diff
@@ -123,6 +183,16 @@ Group::TakeChildren(GroupPtr& aSource) {
   aSource->m.Clear();
 }
 
+void
+Group::SetPreRenderLambda(const RenderLambda& aLambda) {
+  m.preRenderLambda = m.createLambdaDrawable(aLambda);
```

Comment:
Reset the `preRenderLambda` when the aLambda parameter is null function, otherwise null function can be run in the drawable. A user may want to stop running the lambda at some point.

Reply:
Good catch. I meant that do that and forgot.